ironfish/src/migrations/data/000-template/schemaOld.ts

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { PUBLIC_ADDRESS_LENGTH } from '@ironfish/rust-nodejs'
+import bufio from 'bufio'
+import { IDatabase, IDatabaseEncoding, IDatabaseStore, StringEncoding } from '../../../storage'
+
+/* The schemaOld.ts file must define the value schema and database encoding for
+ * ALL datastores that the migration reads from. Even if the migration does not
+ * modify a datastore _A_, if the migration needs to read data from _A_ in order
+ * to write to another datastore _B_, then the schema and encoding for _A_ must
+ * be defined in schemaOld.ts.
+ *
+ * The example below is taken from Migration022, which added the viewKey field
+ * to the AccountValue schema. */
```

Comment:
We can probably simplify:

If you read or write from a store, you will need to have a copy of the schema (example `AccountValue`) and IDatabaseEncoding (example `AccountValueEncoding`) here. Generally for `schemaOld.ts`, you can copy the schema and encoding from the originating file on `staging` branch.

Reply:
That sounds good. The `_A_` and `_B_` isn't very clear

Reply:
Took jow's example, but trimmed the part about staging